pkg/agent/sysadvisor/plugin/inference/inference.go

```diff
@@ -127,3 +163,106 @@ func (infp *InferencePlugin) fetchModelResult(ctx context.Context) {
 
 	wg.Wait()
 }
+
+func (infp *InferencePlugin) syncModelResults(ctx context.Context) {
```

Comment:
not suggest to emit kcmas-related metrics everywhere, maybe it's better to put in metric package instead

Reply:
I agree with you, but why I emit kcmas-metrics is it's hard to reuse podSyncer under metric package.
So do you accept that is goes as:
Create a new file under metric package. And define an exported function(not belongs to any struct). Then call it under `InferencePlugin.Run`?